presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -417,6 +432,11 @@ private static String trimEmptyToNull(String value)
         return emptyToNull(nullToEmpty(value).trim());
     }
 
+    private static Map<String, String> trimEmptyToNull(Map<String, String> properties)
+    {
+        return properties.isEmpty() ? properties : null;
```

Comment:
This looks inverted

Reply:
This function is not used. I'll remove it.